.github/CODEOWNERS

```diff
@@ -64,14 +66,18 @@
 # PRLabel: %App Configuration
 /sdk/appconfiguration/                                               @YalinLi0312
 
+# ServiceOwners: @mrm9084
```

Comment:
```suggestion
# ServiceOwners: @mrm9084
```

Since this is a service-owned library, if we leave this element off, it will infer the service owners based the reviewers.  _(weird, I know)_

Otherwise, we'd want to split this into two blocks:

```text
# ServiceOwners: @mrm9084
# ServiceLabel: %App Configuration Provider

# PRLabel: %App Configuration Provider
/sdk/appconfiguration/azure-appconfiguration-provider/               @mametcal @albertofori @avanigupta @mrm9084
```